downstairs/src/region.rs

```diff
@@ -197,12 +232,36 @@ impl Extent {
         file.seek(SeekFrom::Start(0))?;
 
         /*
-         * Write out the new inner.meta structure for this new extent.
+         * Create the metadata db
          */
+        path.set_extension("db");
+        let metadb = Connection::open(&path)?;
+        assert!(metadb.is_autocommit());
+        metadb.pragma_update(None, "journal_mode", &"WAL")?;
+
+        /*
+         * Create tables and insert base data
```

Comment:
Is there any automatic way to go between the internal ExtentMeta type and the database table, or is that path filled with peril?  If we ever change one, we have to change the other.  Maybe that is something for down the road (if at even possible).

Reply:
We could use an ORM like [Diesel](https://github.com/diesel-rs/diesel) (which supports sqlite) but it may be too heavy for our uses.

Reply:
Yeah, let's not get heavy until we know we need to.